src/Symfony/Component/Mailer/Bridge/Mailchimp/Transport/MandrillApiTransport.php

```diff
@@ -111,7 +111,7 @@ private function getPayload(Email $email, Envelope $envelope): array
                 continue;
             }
 
-            $payload['message']['headers'][] = $name.': '.$header->getBodyAsString();
+            $payload['message']['headers'][$name] = $header->getBodyAsString();
```

Comment:
How does Mandrill handle several headers with the same name? Here you will only ever get the last one.

Reply:
As far as I can tell, Mandrill only allows one of each header in the headers field.